<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Идентификатор текущего процесса</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ID_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Псевдодескриптор текущего процесса</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="p_handle_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Дескриптор, полученный функцией DuplicateHandle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="d_handle_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Дескриптор, полученный функцией OpenProcess</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="op_handle_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Закрываем дескриптор, полученный функцией DuplicateHandle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="close_d_handle_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Закрываем дескриптор, полученный функцией OpenProcess: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="close_op_handle_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
